restful.module

```diff
@@ -197,6 +197,16 @@ function restful_menu() {
 }
 
 /**
+ * Implements hook_entity_info_alter().
+ *
+ * Define the name property as the label of the user. We need this inorder for
+ * users endpoint to be able to autocomplete users.
+ */
+function restful_entity_info_alter(&$entity_info) {
```

Comment:
I don't think we would like to add that - it's too general. I think it's better to override the autocomplete function in the user resource, and making sure it queries by the ``name`` property.